### ISSUE TITLE
create linux distribution packages

### ISSUE THREAD

**Issue by ThomasWaldmann** [infothrill/python-dyndnsc#38]
as getting stuff into linux dists takes a while (esp. until it ends up in a stable release), we should go for this early.

**Comment by infothrill** [comment]
I agree with trying for debian rather than yet another ubuntu ppa. I have not gathered experience building .debs and certainly not getting them included. So the current status from me is: try to put something working in place under the 'packaging/' directory and experiment by using vagrant for virtualization. What I just committed doesn't work yet. Feel free to hack and slice.

**Comment by cben** [comment]
Automated builds: https://build.opensuse.org/ sounds promising?

Sorry, zero experience packaging so can't help more.

**Comment by infothrill** [comment]
Building a python3 only debian package using `stdeb` works out of the box. However, the installed package will quickly fail because debian wheezy (7.x) packages an older version of the requests library (0.12.1).
